controllers/odm.py

```diff
@@ -1,4 +1,4 @@
-from flask import Blueprint, jsonify, request
+from flask import Blueprint, jsonify, request, Response, make_response
```

Comment:
Are you using the ```make_response``` import? It seems like you modified your import statement here, specifically adding ```Response``` and ```make_response``` (which seems like a great idea to help make sure that stuff returned by Flask has all the right headers and so forth), but I don't see where you're actually using it?